src/wt/worker.js

```diff
@@ -1,6 +1,17 @@
-// n should be received from main thread
-export const nthFibonacci = (n) => n < 2 ? n : nthFibonacci(n - 1) + nthFibonacci(n - 2);
-
-export const sendResult = () => {
-    // This function sends result of nthFibonacci computations to main thread
-};
\ No newline at end of file
+import { parentPort, workerData } from "worker_threads";
+
+// n should be received from main thread
+export const nthFibonacci = (n) =>
+  n < 2 ? n : nthFibonacci(n - 1) + nthFibonacci(n - 2);
+
+export const sendResult = async () => {
+  try {
+    if (workerData === 13) throw new Error("shit");

```

Comment:
🤬 
https://blog.codersrank.io/how-often-the-developers-use-swearwords-in-their-code/